### ISSUE TITLE
Button in Product Search block is missing accessible text

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-gutenberg-products-block#1671]
Axe returns an issue when testing the _Product Search_ block:

>  Buttons must have discernible text

It refers to the arrow (`>`) button.

**To Reproduce**
1. Install the Axe extension ([Firefox](https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/), [Chrome](https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd)).
2. Create a post with a _Product Search_ block and run Axe analysis.
3. Notice the `Buttons must have discernible text` error appears.

**Screenshots**
![imatge](https://user-images.githubusercontent.com/3616980/73365524-b19ebd80-42ac-11ea-8d63-ed29ef70b046.png)

This issue is a bit tricky because directly changing the markup invalidates the block.
